stdlib/src/builtin/int_literal.mojo

```diff
@@ -35,13 +35,13 @@ struct IntLiteral(Intable, Stringable, Boolable, EqualityComparable):
     )
 
     @always_inline("nodebug")
-    fn __init__() -> Self:
+    fn __init__(inout self):
         """Default constructor.
 
         Returns:
             The constructed Self object.
         """
-        return Self._zero
+        self._zero
```

Comment:
Not entirely comfortable with this usage, is it correct?

Reply:
No, this isn't correct, you need `self = Self._zero`.